### PR TITLE
Roach food steak

### PR DESCRIPTION
**Pull request by Slepoi**
Made roast from cockroach meat. It turned out very tasty.

### CODE REVIEW

**Comment by RandomColonist** [comment]
Recipe missing reagents such as pepper and salt.

File: code/modules/food/recipes_microwave.dm [hunk 1]
```diff
@@ -943,6 +943,12 @@ I said no!
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/fries
 
+
+/datum/recipe/stakeroach
```
Comment (NanakoAC) [inline code/modules/food/recipes_microwave.dm]: steak not stake

**Comment by LethalGhost** [comment]
> what does this roach steak look like?

It can be viewed now with [Icon Diff](https://github.com/discordia-space/CEV-Eris/pull/2448/checks?check_run_id=46413169)

**Comment by LagannPM** [comment]
He is banned so continuation of this is unlikely

**Pull request closed without merging**